sdk/servicebus/Azure.Messaging.ServiceBus/tests/Diagnostics/EventSourceTests.cs

```diff
@@ -1278,7 +1278,8 @@ private Mock<ServiceBusConnection> GetMockConnection(Mock<TransportReceiver> moc
                     It.IsAny<uint>(),
                     It.IsAny<string>(),
                     It.IsAny<string>(),
-                    It.IsAny<bool>()))
+                    It.IsAny<bool>(),
```

Comment:
Should there be different/additional logging to capture the cross-entity transaction scope in messages - or is that not a helpful thing?

Reply:
I don't think so - we don't do special logging for operations within transactions in general